.github/workflows/compiler-warnings.yml

```diff
@@ -67,3 +67,38 @@ jobs:
       run: |
           cmake -B build -DCMAKE_C_COMPILER=gcc-11 -DCMAKE_CXX_COMPILER=g++-11 -DCMAKE_C_FLAGS="-Werror" -DCMAKE_CXX_FLAGS="-Werror"
           cmake --build build --target ispc -j4
+
+  msvc-2022:
+    runs-on: windows-2022
+    env:
+      LLVM_VERSION: "15.0"
+      LLVM_TAR: llvm-15.0.7-win.vs2019-Release+Asserts-x86.arm.wasm.tar.7z
+      LLVM_HOME: "C:\\projects\\llvm"
+      CROSS_TOOLS_GNUWIN32: "C:\\projects\\cross\\gnuwin32"
+      BUILD_TYPE: "Release"
+      ISPC_OPAQUE_PTR_MODE: "ON"
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Setup MSVC dev cmd
+      uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Would be good to avoid third-party actions from unverified developers.

Would this actions work instead https://github.com/microsoft/setup-msbuild ?

Reply:
OK, moreover neither of them looks needed.